odl_video/settings.py

```diff
@@ -475,6 +475,15 @@ def get_all_config_keys():
 USWITCH_URL = get_string('USWITCH_URL', '')
 LECTURE_CAPTURE_USER = get_string('LECTURE_CAPTURE_USER', '')
 
+# x509 certificate and private key content as environment variables - required
+MIT_WS_CERTIFICATE = get_key('MIT_WS_CERTIFICATE', '')
+MIT_WS_PRIVATE_KEY = get_key('MIT_WS_PRIVATE_KEY', '')
```

Comment:
It looks like these two variables are assigned at lines 405 and 406 as well. Can you get rid of those?